pallets/multisig/src/lib.rs

```diff
@@ -1173,4 +1192,18 @@ mod migration {
             });
         log::info!(" >>> Migrated {} MultiSig Signers.", total_ms_signers);
     }
+
+    pub fn migrate_v2<T: Config>() {
+        sp_runtime::runtime_logger::RuntimeLogger::init();
+
+        log::info!(" >>> Updating MultiSig storage. Migrating MultiSigSigners..");
+        let total_ms_signers =
+            v2::MultiSigSigners::<T>::drain().fold(0usize, |total_ms_signers, (signer, ms, _)| {
+                // Migrate MultiSigSigner.
+                <MultiSigSigners<T>>::insert(&signer, &ms, true);
```

Comment:
The parameter names are backwards.  The `ms` is the first key in the double map and `signer` is the second key.